test/integration/esys-encrypt-decrypt.int.c

```diff
@@ -207,6 +207,12 @@ test_esys_encrypt_decrypt(ESYS_CONTEXT * esys_context)
                     &outPrivate2,
                     &outPublic2,
                     &creationData2, &creationHash2, &creationTicket2);
+
+    if (r == 0x2c2) { /*<< tpm:parameter(2):inconsistent attributes */
+        LOG_WARNING("Unsupported symmetric cipher.");
+        failure_return = EXIT_SKIP;
+        goto error;
```

Comment:
All these goto for 'not supported' cases are redundant.

Reply:
This goto is only for 0x2c2 the goto_if_error triggers for everything else.

Reply:
for everything else including 0x2c2